tutorials/roofit/rf503_wspaceread.py

```diff
@@ -30,9 +30,9 @@
 # -----------------------------------------------------------------
 
 # Retrieve x, and data from workspace
-x = w.var("x")
-model = w.pdf("model")
-data = w.data("modelData")
+x = w("x")
+model = w("model")
+data = w("modelData")
```

Comment:
```suggestion
x = w["x"]
model = w["model"]
data = w["modelData"]
```
The idea was to have the dictionary-like syntax with the bracket operator.